colossalai/inference/tensor_parallel/policies/bloom.py

```diff
@@ -40,6 +41,33 @@ def module_policy(self):
         if not self.shard_config.inference_gptq:
             policy = super().module_policy()
         else:
+            policy[BloomBlock] = ModulePolicyDescription(attribute_replacement={
```

Comment:
Do we need to handle `lm_head` when using gptq? `super().module_policy()` here execute `LlamaPolicy.module_policy` and then `LlamaForCausalLMPolicy.module_policy`. https://github.com/hpcaitech/ColossalAI/blob/ce97790ed73f7962ab1ceae057a020168b45dda4/colossalai/shardformer/policies/llama.py#L202-L209C1

Reply:
now only update decoder block after call uper().module_policy().